packages/odyssey-react-mui/src/Dialog.tsx

```diff
@@ -160,7 +161,13 @@ const Dialog = ({
       </DialogContent>
 
       {actionButtons.length > 0 && (
-        <DialogActions>{actionButtons}</DialogActions>
+        <DialogActions>
+          {actionButtons.map((actionButton, index) => {
```

Comment:
Can we shorten this up a little bit. I know this was my initial example of the fix. I personally like this a little bit better
```js
{actionButtons.map((actionButton, index) =>
  actionButton
    ? cloneElement(actionButton, { key: index })
    : null,
)}```
And then just import `cloneElement` rather than `React`